.travis.yml

```diff
@@ -2,10 +2,7 @@ language: clojure
 script:
     - lein test
     - lein yagni
-    - lein eastwood
+    - lein eastwood '{:exclude-namespaces [api.handler]}'
```

Comment:
if we have to execute eastwood with this options, it's better to have it defined in the `project.clj`

Reply:
I prefere it this way. I think's it's worth that when using `lein eastwood` from our machines we get all warnings from `api.handler`.

Ideally I'd exclude only the warning saying that the api.handler ns is not called, but I don't know how.